cmd/vic-machine/main.go

```diff
@@ -67,7 +67,6 @@ func main() {
 			Usage:  "Update VCH configuration",
 			Action: configure.Run,
 			Flags:  configure.Flags(),
-			Hidden: true,
```

Comment:
Suggestion: add a test similar to https://github.com/vmware/vic/blob/master/tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot#L132 in `6-16-Config.robot`.